apps/server/src/adapters/WebsocketAdapter.ts

```diff
@@ -256,6 +279,29 @@ export class SocketServer implements IAdapter {
     }
   }
 
+  public sendRuntimeStoreUpdate(keys: (keyof RuntimeStore)[], store: Partial<RuntimeStore>) {
```

Comment:
suggestion: we would likely prefer not having distributed logic to achieve a task

ie: the Websocket adapter knows how to send payloads, but shouldnt care whats in it

if we want to make logic specific to the patch payloads, it should be in the store